products/views.py

```diff
@@ -1,3 +1,24 @@
-from django.shortcuts import render
+from django.views import View
+from django.http import JsonResponse
 
-# Create your views here.
+from .models import Product
+
+class ProductSearchView(View):
+    def get(self, request):
+
+        page          = int(request.GET.get('page', 1))
+        limit         = int(request.GET.get('limit', 5))
+        offset        = (page-1) * limit
+        search        = request.GET.get('search', None)
+        products_all  = Product.objects.prefetch_related('image_set').filter(name__icontains = search)
+        count         = products_all.count()
+        products      = products_all[offset:offset + limit]
+        
+        results = [{
+            'id'    : product.id,
+            'name'  : product.name,
+            'price' : int(product.price),
+            'images': product.image_set.all()[0].url
```

Comment:
`images` 복수형 변수를 사용하고 있는데 url 하나를 담고 있네요. 수정해주세요~
그리고 하나만 반환하는 이유가 있나요?

```suggestion
            'images': product.image_set.all()[0].url
```

Reply:
수정했습니다
리스트뷰에서 하나만 필요하다고 해서 사진 하나만 출력될수있도록 로직을 짰습니다.